Maple2.Database/Storage/Game/GameStorage.SystemBanner.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Maple2.Model.Game;
+
+namespace Maple2.Database.Storage;
+
+public partial class GameStorage {
+    public partial class Request {
+        public IList<SystemBanner> GetBanners() {
+            return Context.SystemBanner
+                .AsEnumerable()
```

Comment:
Once you call `AsEnumerable()` the SQL query is finalized and all subsequent logic is executed locally. This means the whole table is going to be returned here. Surely it's possible to have the DB perform the time filter below right?